bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/AddressService.kt

```diff
@@ -148,6 +152,11 @@ class AddressService(
      * - Upserting the new relations
      */
     fun upsertAddresses(addresses: Collection<AddressGateInputRequest>) {
+
+        addresses.forEach { address ->
+            changelogRepository.save(ChangelogEntity(address.externalId, LsaType.Address))
+        }
```

Comment:
Better move the changelog after the actual upsert since the call to CDQ could go wrong